tests/testutil.hpp

```diff
@@ -62,6 +62,8 @@
 #include <stdarg.h>
 #include <string>
 #include <string.h>
+#include <sys/types.h>
```

Comment:
Appveyor seems to be broken at the moment unfortunately - are these headers available with MSVC? I think it's safer to move them below in the !_WIN32 section

Reply:
I have no idea, but should I then put the `getgrgid()` also inside such an `#ifdef`?

Reply:
The IPC tests are already all disabled on Windows, so that's not necessary